src/main/java/duke/Task.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.time.LocalDate;
+
+//Task class to save description of tasks and mark them as done or undone
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    protected LocalDate date;
+
```

Comment:
Similar to previously mentioned, perhaps it may be best to include javadocs comments on the methods in the classes